src/DonorProfiles/resources/js/app/components/tab-menu/utils/index.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+import { getAPIRoot, getAPINonce } from '../../../utils';
+
+export const logoutWithAPI = () => {
+	return axios.post( getAPIRoot() + 'give-api/v2/donor-profile/logout', {
+		headers: {
+			'X-WP-Nonce': getAPINonce(),
+		},
```

Comment:
If you have something consistent like this that needs to be done with _every_ axios request, then it's handy to use the [axios.create](https://github.com/axios/axios#axioscreateconfig), which then gets used by all of the places you're otherwise using axios.

Reply:
This is a good idea -- I'm gonna hold off on implementing for a future code polish PR, since I'd rather have consistent implementation across the domain (which we currently have) over introducing this syntactical sugar here only.